moveos/moveos-stdlib/src/natives/moveos_stdlib/any_table/mod.rs

```diff
@@ -0,0 +1,665 @@
+// Copyright (c) RoochNetwork
+// Copyright (c) The Diem Core Contributors
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+/// A native Table implementation for save any type of value.
+/// Refactor from https://github.com/rooch-network/move/blob/5b413d009515ad1144042ded27cbe9bd702aaad7/language/extensions/move-table-extension/src/lib.rs#L4
```

Comment:
any table is used to implement storage abstraction?

Reply:
yes